core/src/main/java/org/apache/calcite/plan/Strong.java

```diff
@@ -264,6 +264,7 @@ private boolean anyNull(List<RexNode> operands) {
     map.put(SqlKind.LEAST, Policy.ANY);
     map.put(SqlKind.TIMESTAMP_ADD, Policy.ANY);
     map.put(SqlKind.TIMESTAMP_DIFF, Policy.ANY);
+    map.put(SqlKind.ITEM, Policy.ANY);
```

Comment:
shall we need to add a test case for ITEM in RexProgramTest for checking the strong of item?

Reply:
Yes, we need that.

Reply:
Makes sense .. I will add this test.  Thanks for the suggestion.